projects/plugins/jetpack/modules/google-fonts/current/class-jetpack-google-font-face.php

```diff
@@ -52,9 +52,11 @@ public function current_screen() {
 
 	/**
 	 * Print fonts that are used in global styles or block-level settings.
+	 *
+	 * @todo Remove phan suppression when the WP_Font_Face_Resolver class is available in the minimum WordPress version.
 	 */
 	public function print_font_faces() {
-		$fonts             = WP_Font_Face_Resolver::get_fonts_from_theme_json();
+		$fonts             = WP_Font_Face_Resolver::get_fonts_from_theme_json(); // // @phan-suppress-current-line PhanUndeclaredClassMethod
```

Comment:
```suggestion
		$fonts             = WP_Font_Face_Resolver::get_fonts_from_theme_json(); // @phan-suppress-current-line PhanUndeclaredClassMethod
```

Reply:
Also, if this method might not be available, shouldn't we wrap it in a check to prevent errors?

We could also do the same for `wp_print_font_faces()`, but really we should abort prior to that point.

Reply:
I didn't since the include is further up the stack after a version check.